internal/controller/zz_setup.go

```diff
@@ -21,25 +21,25 @@ import (
 
 	"github.com/crossplane/terrajet/pkg/controller"
 
+	role "github.com/crossplane-contrib/provider-jet-datadog/internal/controller/datadog/role"
 	downtime "github.com/crossplane-contrib/provider-jet-datadog/internal/controller/downtime/downtime"
 	json "github.com/crossplane-contrib/provider-jet-datadog/internal/controller/monitor/json"
 	monitor "github.com/crossplane-contrib/provider-jet-datadog/internal/controller/monitor/monitor"
 	servicelevelobjective "github.com/crossplane-contrib/provider-jet-datadog/internal/controller/monitor/servicelevelobjective"
 	providerconfig "github.com/crossplane-contrib/provider-jet-datadog/internal/controller/providerconfig"
-	role "github.com/crossplane-contrib/provider-jet-datadog/internal/controller/role/role"
 	test "github.com/crossplane-contrib/provider-jet-datadog/internal/controller/synthetics/test"
 )
 
 // Setup creates all controllers with the supplied logger and adds them to
 // the supplied manager.
 func Setup(mgr ctrl.Manager, o controller.Options) error {
 	for _, setup := range []func(ctrl.Manager, controller.Options) error{
+		role.Setup,
 		downtime.Setup,
 		json.Setup,
 		monitor.Setup,
 		servicelevelobjective.Setup,
 		providerconfig.Setup,
-		role.Setup,
```

Comment:
Nit: why change the alphabetic order here?

Reply:
Didn't think about alphabetic order, I just appended :)

Reply:
No, wait. That file is auto generated. I don't control that ?

Reply:
Maybe it lists resources with no short group first ?